customize_image.sh

```diff
@@ -390,6 +390,12 @@ cat <<EOL >$IMAGEDIR/etc/NetworkManager/conf.d/99-unmanaged-devices.conf
 unmanaged-devices=interface-name:pileft;interface-name:piright
 EOL
 
+# Use fallback to dhcp if no connection is configured
+cp "$BAKERYDIR/templates/network-manager/fallback-dhcp.nmconnection" "$IMAGEDIR/etc/NetworkManager/system-connections"
```

Comment:
Use [install](https://www.man7.org/linux/man-pages/man1/install.1.html) instead of cp